### ISSUE TITLE
VMS: Youth Volunteer Vacation Apply Button

### ISSUE THREAD

**Issue by ebskopin** [washingtontrails/vms#551]
For youth volunteer vacations, we would like the "Join Work Party" button to read "Apply" and link directly to the youth application - http://www.wta.org/volunteer/youth/youth/Youth-Volunteer-Vacations-Application

People interested in attending these events will not access the normal registration processes, and will instead be registered by staff via Salesforce.

**Comment by davisagli** [comment]
How do we tell if the work party is a youth volunteer vacation? When the Work Party Type's name equals "Youth Vacation"? I see there is also a work party type in production SF called "Youth Vacation - backcountry", so maybe it should be when the Work Party Type's name *starts with* "Youth Vacation"?

I assume that if the user is logged in as someone who was already registered by staff in Salesforce, they should continue to see the normal status message with the option to cancel. In other words this change is only for users who aren't registered yet. Correct?

**Comment by ebskopin** [comment]
Correct, when WP type equals "Youth Vacation." I have removed the "Youth Vacation - backcountry" wp type in production. All will be under on moniker, "Youth Vacation."

Yes to all that you said in your second paragraph.

**Comment by davisagli** [comment]
Okay, you can try this out on staging. I also renamed the "Youth Volunteer Vacation" work party type to "Youth Vacation" so it's the same as in production SF.

**Comment by ebskopin** [comment]
Excellent! Thank-you David.  1 quick mod - could you change the button to read "Apply Now", please. Otherwise this looks great.
